enterprise/server/raft/cache/cache.go

```diff
@@ -546,3 +623,89 @@ func (rc *RaftCache) SupportsCompressor(compressor repb.Compressor_Value) bool {
 func (rc *RaftCache) SupportsEncryption(ctx context.Context) bool {
 	return false
 }
+
+func (rc *RaftCache) olderThanThreshold(t time.Time, threshold time.Duration) bool {
```

Comment:
not sure this func adds much value (and it's only called once)

Reply:
removed.